lib/call-cabal-project-to-nix.nix

```diff
@@ -21,8 +21,8 @@ in
 , caller               ? "callCabalProjectToNix" # Name of the calling funcion for better warning messages
 , ghc           ? null # Deprecated in favour of `compiler-nix-name`
 , ghcOverride   ? null # Used when we need to set ghc explicitly during bootstrapping
-, nix-tools     ? evalPackages.haskell-nix.nix-tools.${compiler-nix-name}     # When building cabal projects we use the nix-tools
-, cabal-install ? evalPackages.haskell-nix.cabal-install.${compiler-nix-name} # and cabal-install compiled with matching ghc version
+, nix-tools     ? evalPackages.haskell-nix.nix-tools-unchecked.${compiler-nix-name}     # When building cabal projects we use the nix-tools
```

Comment:
Why do we need the unchecked version now?

Reply:
The unchecked versions fall back on ghc865 when the materialized files are not present.